Model/Cron.php

```diff
@@ -119,6 +119,11 @@ class Cron
      */
     protected $_acquirerReference;
 
+    /**
+     * @var
+     */
+    protected $ratepayDescriptor;
```

Comment:
minor: shall we keep the convention of underscore ($_ratepayDescriptor) ?

Reply:
Let's use the PSR-2: Coding Style Guide and there it is specific mentioned to not using the underscore anymore see: https://www.php-fig.org/psr/psr-2/


quote:
```
Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.
```